SPECS/ocaml-findlib/ocaml-findlib.spec

```
@@ -64,39 +85,33 @@ make install \
      prefix=%{buildroot} \
      OCAMLFIND_BIN=%{_bindir} \
      OCAMLFIND_MAN=%{_mandir}
+ 
+# Remove spurious executable bits
+chmod 0644 %{buildroot}%{_mandir}/man{1,5}/*
+chmod 0644 %{buildroot}%{_libdir}/ocaml/findlib/*.{cma,cmi,ml,mli}
+chmod 0644 %{buildroot}%{_libdir}/ocaml/findlib/{META,Makefile*}
+%ifarch %{ocaml_native_compiler}
+chmod 0644 %{buildroot}%{_libdir}/ocaml/findlib/*.{a,cmxa}
+%endif
 
+%ocaml_files
+sed -i '/ocamlfind\.conf/d' .ofiles
 
-%files
+%files -f .ofiles
 %license LICENSE
 %doc doc/README
 %config(noreplace) %{_sysconfdir}/ocamlfind.conf
-%{_bindir}/*
-%{_mandir}/man1/*
-%{_mandir}/man5/*
-%{_libdir}/ocaml/*/META
-%{_libdir}/ocaml/topfind
-%{_libdir}/ocaml/findlib
-%ifarch %{ocaml_native_compiler}
-%exclude %{_libdir}/ocaml/findlib/*.a
-%exclude %{_libdir}/ocaml/findlib/*.cmxa
-%endif
-%exclude %{_libdir}/ocaml/findlib/*.mli
-%exclude %{_libdir}/ocaml/findlib/Makefile.config
-%exclude %{_libdir}/ocaml/findlib/make_wizard
-%exclude %{_libdir}/ocaml/findlib/make_wizard.pattern
-# Had to disable this in OCaml 4.06, unclear why.
-#%%{_libdir}/ocaml/num-top
-
-%files devel
-%doc doc/README doc/guide-html
-%ifarch %{ocaml_native_compiler}
-%{_libdir}/ocaml/findlib/*.a
-%{_libdir}/ocaml/findlib/*.cmxa
-%endif
-%{_libdir}/ocaml/findlib/*.mli
-%{_libdir}/ocaml/findlib/Makefile.config
+ 
+%files devel -f .ofiles-devel
+%doc LICENSE 
```

Another doc vs license issue.
```suggestion
%license LICENSE
```

Since this is a `devel` package, which pulls in the default one with its license, you can also do this:
```suggestion
```